linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/query.js

```diff
@@ -68,7 +68,7 @@ var Query = function() {
     }
     q.withClients = function(clients) {
       if (_.isArray(query.clientLabels))
-        query.clientLabels.concat(clients);
+        query.clientLabels = query.clientLabels.concat(clients);
       return q;
     }
```

Comment:
they're there for library completeness/symmetry, I expect they'll be used at some point